manifests/v/VirtualHere/USBClient/5.4.3/VirtualHere.USBClient.installer.yaml

```
@@ -0,0 +1,22 @@
+# Automatically updated by the winget bot at 2023/Feb/13
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: VirtualHere.USBClient
+PackageVersion: 5.4.3
+MinimumOSVersion: 10.0.0.0
+InstallerType: portable
+Commands:
+- vhclient
+ReleaseDate: 2022-10-13

```

```suggestion
ReleaseDate: 2022-10-13
```
Incorrect ReleaseDate